dapps/tests/contracts/test/ownable_test.sol

```diff
@@ -12,6 +12,6 @@ contract OwnableTests {
   }
 
   function shouldnotbezeroAddress() public {
-    Assert.equal(true, true, "owner is uninitialized");
+    require(true == true, "owner is uninitialized");
```

Comment:
Why are we making this change?

Reply:
We had solidity errors because `Assert.` didn't exist. 
We had forgotten that those are for the solidity tests. Eric reminded me yesterday. I'll need to revert and find why the remix tests don't work